src/client/service-worker/events/utility.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import {
+  match, put, remove, getMetaData, setMetaData, createCacheName,
+} from '@americanexpress/one-service-worker';
+
+const localeRegExp = /(([a-z]{2,3})(?:-)?([a-zA-Z]{1,})?)\/[^/]*\.json/;
```

Comment:
https://www.npmjs.com/package/safe-regex shows as unsafe, please test thoroughly.

Reply:
When it comes to safety concerns, let's establish that this is a client-side app and any attacks on the RegExp would only affect the attacker at this point. The RegExp has three capture groups and no forward/backward looking behavior - I will tighten it up and update with that.

The library does state "WARNING: This module has both false positives and false negatives. Use `vuln-regex-detector` for improved accuracy." which may not be accurate.

Reply:
Simplified the regexps in efd5586